src/test/java/Homework18.java

```diff
@@ -0,0 +1,38 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+    @Test
+
+    public void addSongToPlaylist() throws InterruptedException {
+        String newSongAddedToPlaylistNotification = "Added 1 song into";
+        //Opens url WebPage
+        accessUrlPage();
+        //Enters email credentials into webpage Email space
+        provideEmailCredentials("james.patterson@testpro.io");
+        providePasswordCredentials("te$t$tudent");
+
+        loginButton();
+
+//        songSearch("Pluto");
+//
+//        viewAllSongs();
+//
+//        firstSongSelection();
+//
+//        addToButton();
+//
+//        selectedSongAddedToPlaylist();
+//
+//        Assert.assertTrue(addedSongMessage().contains(newSongAddedToPlaylistNotification));
+//
+//        homeButton();
+
+//        playSong();
+
+        clickPlayButton();
+
+        Assert.assertTrue(songPlaying());
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```
For example

![image](https://github.com/testprojp/A47-QA-Automation-Framework/assets/104761482/f196919b-de24-4cba-bcb2-73df066aa9f4)